src/dash/student_view/TutorReviewModal.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+class TutorModal extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            first_star: false,
+            second_star:false,
+            third_star:false,
+            fourth_star:false,
+            fifth_star:false,
+            rating: 0,
+            satisfaction: '',
+            error_message:'hide'
+        }
+        this.changeStar = this.changeStar.bind(this);
+        this.setRating = this.setRating.bind(this);
+        this.changeStarOut = this.changeStarOut.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    changeStar(number) {
+        if (number == 1) {
+            this.setState({first_star:true,second_star:false,third_star:false,fourth_star:false,fifth_star:false});
+        } else if (number == 2) {
+            this.setState({first_star:true,second_star:true,third_star:false,fourth_star:false,fifth_star:false});
```

Comment:
Interesting choice of state representation... Any reason we couldn't just do this with the number passed into the function?